frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/date_cell/date_editor.dart

```diff
@@ -49,7 +49,7 @@ class _DateCellEditor extends State<DateCellEditor> {
       ),
       builder: (BuildContext context, snapshot) {
         if (snapshot.hasData) {
-          return _buildWidget(snapshot);
+          return  _buildWidget(snapshot) ;
```

Comment:
```suggestion
          return _buildWidget(snapshot);
```

No need for these whitespace changes.